html/legacy_index.php

```diff
@@ -303,17 +303,17 @@ function popUp(URL)
 }
 
 if (is_array($msg_box)) {
-    echo("<script>
+    echo "<script>
         toastr.options.timeout = 10;
         toastr.options.extendedTimeOut = 20;
-    ");
+        </script>
+    ";
+
     foreach ($msg_box as $message) {
-        $message['type'] = mres($message['type']);
-        $message['message'] = mres($message['message']);
-        $message['title'] = mres($message['title']);
-        echo "toastr.".$message['type']."('".$message['message']."','".$message['title']."');\n";
+        Toastr::add($message['type'], $message['message'], $message['title']);
```

Comment:
Is this working for you? I can't seem to get it working.

`FORCE=1 ./composer.phar remove wpb/string-blade-compiler`

I would have expected that to have led to a toastr notification but I don't see anything.

Reply:
@laf you have to revert composer.json after removing it to get a validation error.
I did get a toastr after removing the package.